client/src/components/Contact.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class Contact extends React.Component {
+      render() {
+            return (
+                  <div>
+                        <div id="contactmargin">
+                        </div>
+                        <div id="contentflex">
+                              <div>
+                                    <p><img id="contactpic" alt="Emily Saber Lake Champlain" src="/contact.jpg" /></p>
+                                    <p><br></br>EMILY SABER <br></br>BOSTON, MA<br></br> emilysaber13@gmail.com</p>
+                              </div>
+                              <div id="contactpara"><h2>I'D LOVE TO HEAR FROM YOU!</h2>
+                                    <hr></hr>
+                                    <form>
+                                          <ul>
+                                                <li> <input id="text" type='text' placeholder="Name" name='name' /></li>
+                                                <li><input id="text" type='email' placeholder="Email" name='email' /> </li>
+                                                <li> <textarea id="message" name="message" placeholder="Message" /></li>
+                                                <li><input id="submit" type='submit' /></li>
+                                          </ul>
+                                    </form>
```

Comment:
Now that we've gone through servers and using API end points you can get this to send emails to you! They are a little tricky but a really fun feature to have under your belt